components/tests/ui/plugins/robot.py

```diff
@@ -68,11 +68,13 @@ def config(self, args):
         d["WEBPREFIX"] = static_prefix
         d["QWEBPREFIX"] = urllib.quote(static_prefix, '')
         d["QSEP"] = urllib.quote('/', '')
-        if settings.APPLICATION_SERVER in settings.FASTCGI_TYPES:
-            d["WEBHOST"] = d["HOST"]
+        app_host = getattr(settings, 'APPLICATION_SERVER_HOST', None)
+        app_port = getattr(settings, 'APPLICATION_SERVER_PORT', None)
+        if app_host:
+            d["WEBHOST"] = ("%s:%s" %
+                            (app_host, app_port)) if app_port else app_host
```

Comment:
But we don't want the port here only the `application_server.host` - see https://ci.openmicroscopy.org/job/OMERO-DEV-merge-robotframework/129/console